src/hotspot/share/gc/parallel/psCompactionManager.hpp

```diff
@@ -38,14 +39,15 @@ class ParallelCompactData;
 class ParMarkBitMap;
 
 class ParCompactionManager : public CHeapObj<mtGC> {
+  friend class CompactionWithStealingTask;
```

Comment:
`CompactionWithStealingTask` and `UpdateAndFillClosure` are dead code.

Reply:
I will remove them.